geeutils/eu.py

```diff
@@ -22,14 +22,15 @@
 import ee
 
 
-def authenticate() -> None:
+def authenticate(allow_interactive=True) -> None:
     """Authenticate with the GEE API.
 
     A SERVICE_ACCOUNT_KEY environment variable must be set with the key to a
     Google Cloud Platform service account that has permissions in GEE.
 
     Args:
-        None
+        allow_interactive: If False, does not allow the interactive login workflow.
+                           If True, continue to interactive login workflow.
```

Comment:
Maybe add "(default True)" so users know

Reply:
taken care of.